woocommerce/woocommerce
# CYS: Improve tracking survey

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR is necessary to improve how we track the survey answers. Please for more details, check this private link: PCYsg-PWZ-p2



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Remove these code lines: https://github.com/woocommerce/woocommerce/blob/5d936bb547706a4fe1f2c24f7dda7a418d5fe814/packages/js/tracks/src/index.ts/#L52-L55 (or build a WooCommerce production build)
4. Go to `wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store`.
5. Click on `Design your homepage`.
6. Add a pattern.
8. Click save.
9. Click on share feedback.
10. Fill the survey
11. Open the network tools filter.
12. Submit the survey
13. Search `t.gif` in the network tools filter.
14. Ensure that `ces_feedback` event is tracked.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

CYS: Improve tracking survey

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
